src/containers/ConceptPage/components/ConceptForm.jsx

```diff
@@ -315,52 +335,27 @@ class ConceptForm extends Component {
                   </AccordionWrapper>
                 )}
               </Accordion>
-              <Field right>
-                {error && <span className="c-errorMessage">{error}</span>}
-                <AlertModal
-                  show={showResetModal}
-                  text={t('form.resetToProd.modal')}
-                  actions={[
-                    {
-                      text: t('form.abort'),
-                      onClick: () => this.setState({ showResetModal: false }),
-                    },
-                    {
-                      text: 'Reset',
-                      onClick: () => this.onReset(setValues),
-                    },
-                  ]}
-                  onCancel={() => this.setState({ showResetModal: false })}
-                />
-
-                {inModal ? (
-                  <FormikActionButton outline onClick={onClose}>
-                    {t('form.abort')}
-                  </FormikActionButton>
-                ) : (
-                  <FormikActionButton
-                    onClick={history.goBack}
-                    outline
-                    disabled={isSubmitting}>
-                    {t('form.abort')}
-                  </FormikActionButton>
-                )}
-
-                <SaveButton
-                  {...formClasses}
-                  isSaving={isSubmitting}
+              {
+                <EditorFooter
+                  t={t}
+                  isSubmitting={isSubmitting}
                   formIsDirty={formIsDirty}
-                  showSaved={savedToServer && !formIsDirty}
-                  submit={!inModal}
-                  onClick={evt => {
-                    if (inModal) {
-                      evt.preventDefault();
-                      submitForm();
-                    }
-                  }}>
-                  {t('form.save')}
-                </SaveButton>
-              </Field>
+                  savedToServer={savedToServer}
+                  values={values}
+                  error={error}
+                  errors={errors}
+                  getEntity={this.getConcept}
+                  entityStatus={concept.status}
+                  createMessage={createMessage}
+                  showSimpleFooter={!concept.id}
+                  setFieldValue={setFieldValue}
+                  onSaveClick={() => {
```

Comment:
usikker på denne, LearningResourceForm og TopicArticleForm sender med "saveAsNewVersion" som parameter i denne funksjonen og bruker det i handleSubmit i forbindelse med slate, vi har ikke noe slate i concepts, så jeg vet ikke hvordan det er med newVersion i concepts

Reply:
jeg antar at det har noe med den ekstra knappen ved lagre-knappen å gjøre, med vet ikke hva flyten der skal være

Reply:
Det er for å ha alternative lagringsmetoder. For forklaringer er ikkje det aktuellt så der skal kun vanlig lagreknapp brukes.